Dockerfile.tor-onionbalance-manager

```diff
@@ -1,22 +1,30 @@
-FROM --platform=$BUILDPLATFORM golang:1.17 as builder
+# build
+FROM --platform=$BUILDPLATFORM docker.io/library/golang:1.17-alpine as builder
 
 WORKDIR /src
 
+COPY . /src
+
 # Build
 ARG TARGETOS TARGETARCH
-RUN --mount=target=. \
-    --mount=type=cache,target=/root/.cache/go-build \
+RUN --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=cache,target=/go/pkg \
     CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -o /out/onionbalance-local-manager ./agents/onionbalance/main.go
 
-FROM alpine:3.16.2
+# package build
+FROM docker.io/library/alpine:3.17
+
+ARG OB_VERSION="0.2.2"
 
-ARG VERSION=0.2.2
+RUN apk add --update --no-cache \
+    git python3 py3-pip py3-wheel py3-cryptography py3-setproctitle py3-pycryptodomex
+RUN python3 -m pip --no-cache-dir install git+https://gitlab.torproject.org/tpo/core/onionbalance.git@${OB_VERSION}
 
-RUN apk add --update git python3 py3-pip py3-wheel py3-cryptography py3-setproctitle py3-pycryptodomex
-RUN python3 -m pip install git+https://gitlab.torproject.org/tpo/core/onionbalance.git@${VERSION}
+WORKDIR /app
+RUN chmod g+w .
```

Comment:
`RUN chmod g+w /app`
Don't use relative path if possible

Reply:
I'll rebase my branch and make the recommended updates